texastribune/queso-ui
# Sponsor content updates

#### What's this PR do?
Adds sponsor content updates

##### Classes added (if any)
- c-sponsor-block
##### Classes removed (if any)
- c-sponsor-card

^ We like to call cards blocks so just a rename there


#### Why are we doing this? How does it help us?

The "info-drop" is an evolving element in the sponsor content strategy. This just brings in the latest changes from user testing.

#### How should this be manually tested?
`yarn dev`

See: [c-sponsor-block](http://localhost:3000/pages/components/c-sponsor-block/raw-preview.html)
See: [c-info-block](http://localhost:3000/pages/components/c-info-drop/raw-preview.html)


#### Does this introduce a breaking change where queso-ui is used in the wild? If so, is there a relevant branch/PR to accompany this release?

Nope, I made some of the padding helpers I added a while back more specific to where the padding is being added and globally removed the browser arrow in `details`/`summary` elements.

#### TODOs / next steps:
